src/test/java/no/unit/nva/model/contexttypes/ReportTest.java

```diff
@@ -35,15 +35,15 @@
     @DisplayName("Reports can be created")
     @ParameterizedTest
     @CsvSource({
-        "Series,123,A publisher,9780201309515,0363-6941,1945-662X",
-        "Series,123,A publisher,9780201309515|9788131700075,0363-6941,1945-662X",
-        "Series,123,A publisher,9780201309515|9788131700075,0363-6941,1945-662X",
-        "Series,123,A publisher,9780201309515|9788131700075,0363-6941,1945-662X",
-        "Series,,A publisher,9780201309515|9788131700075,0363-6941,1945-662X",
+        "A Report,123,A publisher,9780201309515,0363-6941,1945-662X",
```

Comment:
Could we replace the "A Report" with "A Report series". Now it is a bit confusing, because we have a a Report belonging to a series called "A Report" and these tests are quite complicated to begin with